lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/head.html

```diff
@@ -1 +1,18 @@
 {% extends 'ace_common/edx_ace/common/base_head.html' %}
+{% block additional_styles %}
```

Comment:
some email clients strip out all style elements outside of head so moving these here